### PR TITLE
[ODS-4663] Fix Initdev and Deployment Scripts to Use Plugin Folder Setting

### PR DESCRIPTION
**Pull request by tbmeador-dlp**

### CODE REVIEW

File: logistics/scripts/modules/settings/settings-management.psm1 [hunk 1]
```diff
@@ -337,9 +433,9 @@ function Add-TestHarnessSpecificAppSettings([hashtable] $Settings = @{ }, [strin
 
 function Get-UserSecretsIdByProject {
     return @{
-        "Application/EdFi.Ods.SandboxAdmin" = "f1506d66-289c-44cb-a2e2-80411cc690ea"
-        "Application/EdFi.Ods.SwaggerUI" = "f1506d66-289c-44cb-a2e2-80411cc690eb"
-        "Application/EdFi.Ods.WebApi" = "f1506d66-289c-44cb-a2e2-80411cc690ec"
+        "Application/EdFi.Ods.SandboxAdmin"               = "f1506d66-289c-44cb-a2e2-80411cc690ea"
+        "Application/EdFi.Ods.SwaggerUI"                  = "f1506d66-289c-44cb-a2e2-80411cc690eb"
+        "Application/EdFi.Ods.WebApi"                     = "f1506d66-289c-44cb-a2e2-80411cc690ec"
         "Application/EdFi.Ods.Api.IntegrationTestHarness" = "f1506d66-289c-44cb-a2e2-80411cc690ed"
```
Comment (blmeyers) [inline logistics/scripts/modules/settings/settings-management.psm1:438]: This is actually a little concerning, each project should have its own unique id, they shouldn't be sharing a secrets file.

**Review by blmeyers — CHANGES_REQUESTED** [review]

File: logistics/scripts/modules/settings/settings-management.psm1 (hunk 1 above)
Comment (tbmeador-dlp) [reply logistics/scripts/modules/settings/settings-management.psm1:438]: I read it the same way the first time, but the last letter of GUID is unique

File: Application/SolutionScripts/InitializeDevelopmentEnvironment.psm1 [hunk 2]
```diff
@@ -114,18 +114,18 @@ function Initialize-DevelopmentEnvironment {
 
         $settings = @{
             ApiSettings = @{
-                Mode                    = $InstallType
-                OdsTokens               = $OdsTokens
-                Engine                  = $Engine
-                UsePlugins              = $UsePlugins.IsPresent
                 MinimalTemplateSuffix   = 'Ods_Minimal_Template'
                 PopulatedTemplateSuffix = 'Ods_Populated_Template'
```
[the remark this answers is not in the document]
Comment (tbmeador-dlp) [reply Application/SolutionScripts/InitializeDevelopmentEnvironment.psm1]: moved to settings

File: logistics/scripts/modules/settings/settings-management.psm1 (hunk 1 above)
Comment (blmeyers) [reply logistics/scripts/modules/settings/settings-management.psm1:438]: Well look at that... these were obviously hand crafted.

**Review by blmeyers — APPROVED** [review]

**Pull request merged**